enumflags/src/fallible.rs

```diff
@@ -25,3 +33,26 @@ macro_rules! impl_try_from {
 impl_try_from! {
     u8, u16, u32, u64, usize
 }
+
+#[derive(Debug, Copy, Clone)]
+pub struct FromBitsError<T: RawBitFlags> {
+    flags: BitFlags<T>,
+    invalid: T::Type,
+}
+
+impl<T: RawBitFlags> FromBitsError<T> {
+    pub fn truncate(self) -> BitFlags<T> {
+        self.flags
+    }
+
+    pub fn invalid_bits(self) -> T::Type {
+        self.invalid
+    }
+}
```

Comment:
Since this is a public API, it'd be nice to have a sentence or two of documentation here. You could even write a doctest which tries to convert a number with some invalid flags and then asserts about the results of `truncate` and `invalid_bits`. This would kill two birds with one stone, since there aren't really any tests for this functionality right now.